hackclub.com.yaml

```diff
@@ -711,6 +711,10 @@ miracosta:
   ttl: 1
   type: CNAME
   value: miracosta.netlify.app.
+msl:
+- ttl: 1
+  type: CNAME
+  value: msl.hackclub.com.
```

Comment:
I've been asking if you're sure this is the correct config, because this won't work... it needs to point to an external CNAME like `miracosta.netlify.app.`